ts/store/reducers/backendStatus.ts

```diff
@@ -353,35 +351,66 @@ export const barcodesScannerConfigSelector = createSelector(
 );
 
 /**
- * Return the remote config for the PN feature.
+ * Return the remote config about PN enabled/disabled
+ * if there is no data, false is the default value -> (PN disabled)
  */
-export const PnConfigSelector = createSelector(
+export const isPnEnabledSelector = createSelector(
   backendStatusSelector,
-  (backendStatus): PnConfig =>
+  (backendStatus): boolean =>
     pipe(
       backendStatus,
-      O.map(bs => bs.config.pn),
-      O.getOrElseW(() => ({
-        enabled: false,
-        frontend_url: "",
-        optInServiceId: ""
-      }))
+      O.map(bs => bs.config.pn.enabled),
+      O.getOrElseW(() => false)
     )
 );
 
 /**
- * Return the remote config about PN enabled/disabled
- * if there is no data or the local Feature Flag is disabled,
- * false is the default value -> (PN disabled)
+ * Return false if the app needs to be updated in order to use PN.
  */
-export const isPnEnabledSelector = createSelector(
+export const isPnSupportedSelector = createSelector(
   backendStatusSelector,
   (backendStatus): boolean =>
-    pnEnabled &&
     pipe(
       backendStatus,
-      O.map(bs => bs.config.pn.enabled),
-      O.getOrElseW(() => false)
+      O.map(bs =>
+        isVersionSupported(
+          Platform.OS === "ios"
+            ? bs.config.pn.min_app_version.ios
+            : bs.config.pn.min_app_version.android,
+          getAppVersion()
+        )
+      ),
+      O.getOrElse(() => false)
+    )
+);
+
+/**
+ * Return the minimum app version required to use PN.
+ */
+export const pnMinAppVersionSelector = createSelector(
+  backendStatusSelector,
+  backendStatus =>
+    pipe(
+      backendStatus,
+      O.map(bs =>
+        Platform.OS === "ios"
+          ? bs.config.pn.min_app_version.ios
+          : bs.config.pn.min_app_version.android
+      ),
+      O.getOrElse(() => "-")
+    )
+);
+
+/**
+ * Return the url of the PN frontend.
+ */
+export const pnFrontendUrlSelector = createSelector(
```

Comment:
Since this is a direct read with minor data creation, should we remove the `createSelector`?